src/identity/components/GlobalHeader/GlobalHeader.tsx

```diff
@@ -28,6 +28,7 @@ import {
   emptyOrg,
 } from 'src/identity/components/GlobalHeader/DefaultEntities'
 import {alphaSortSelectedFirst} from 'src/identity/utils/alphaSortSelectedFirst'
+import IdentityUserAvatar from 'src/identity/components/GlobalHeader/IdentityUserAvatar'
```

Comment:
I called it `IdentityUserAvatar` because it takes in `IdentityUser` object.

Reply:
LGTM.